src/libraries/System.Data.Odbc/ref/System.Data.Odbc.cs

```diff
@@ -99,6 +99,9 @@ public sealed partial class OdbcConnection : System.Data.Common.DbConnection, Sy
         public new System.Data.Odbc.OdbcCommand CreateCommand() { throw null; }
         protected override System.Data.Common.DbCommand CreateDbCommand() { throw null; }
         protected override void Dispose(bool disposing) { }
+        public override System.Data.DataTable GetSchema() { throw null; }
+        public override System.Data.DataTable GetSchema(string collectionName) { throw null; }
+        public override System.Data.DataTable GetSchema(string collectionName, string[] restrictionValues) { throw null; }
```

Comment:
Why are we adding public APIs?

Reply:
We were not exposing these properties, running generateReferenceSource added these missing apis.

Reply:
These are overrides, so ApiCompat doesn't enforce overrides, but as part of the nullability effort we started preserving overrides in ref assemblies, so basically this is just contributing to have up-to-date ref assemblies, but the user should be able to see these already because they're exposed by it's parent class, so it doesn't matter if you compile against the parent exposed ones or  the overrides.